<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桥梁基本状态计算</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>桥梁长度</string>
   </property>
   <widget class="QLineEdit" name="lineLength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入桥梁长度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConfirm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入桥梁的长度: (m)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>约束与作用力</string>
   </property>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>三约束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>二约束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>一约束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonForce_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直力</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonForce_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>均载荷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonForce_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>力矩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonForce_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>水平力</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLength_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>力或载荷或力矩：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>距离1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLength_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>距离2：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLength_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>9</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>桥梁状态</string>
   </property>
   <widget class="QPushButton" name="pushButtonCalculate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>一个简单的轴力剪力弯矩三图的计算软件</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>所有的单位都为基本单位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在添加约束与作用力时，请先在编辑框&lt;/p&gt;&lt;p&gt;输入，再点击按钮进行添加&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加约束时，只需要在“距离1“框中输入&lt;/p&gt;&lt;p&gt;约束所在的位置即可&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>只有一个位置的作用力默认在“距离1”</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>作用力默认向上或顺时针为正</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>距离默认从左到右，从0开始</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>390</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>--Made BY HUST GHR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>桥梁各类图</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>轴力图：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>剪力图：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弯矩图：</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_zhou" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_jian" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="widget_wan" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
